queries/wing/highlights.scm

```diff
@@ -0,0 +1,94 @@
+; Classes
+
+(custom_type) @type
+(class_field 
+  name: (identifier) @field
+) 
+(class_definition 
+  name: (identifier) @type
+)
+(method_definition
+  name: (identifier) @method
+)
+(inflight_method_definition
+  name: (identifier) @function
+)
+
+; Functions
+
+(keyword_argument_key) @parameter
+(call 
+  caller: (reference 
+  	(nested_identifier 
+    	property: (member_identifier) @method.call)) 
+)
+(call 
+  caller: (reference 
+  	(reference_identifier) @method.call)
+)
+
+; Primitives
+
+[
+ (number)
+ (duration)
+] @constant.builtin
+(string) @string
+(bool) @constant.builtin
+(builtin_type) @type.builtin
+(json_container_type) @type.builtin
+
+; Special
+
+(comment) @comment
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+]  @punctuation.bracket
+
+[
+  "-"
+  "+"
+  "*"
+  "/"
+  "%"
+  "<"
+  "<="
+  "="
+  "=="
+  "!"
+  "!="
+  ">"
+  ">="
+  "&&"
+  "??"
+  "||"
+] @operator
+
+[
+  ";"
+  "."
+  ","
+] @punctuation.delimiter
+
+[
+  "as"
+  "bring"
+  "class"
+  "else"
+  "for"
+  "if"
+  "in"
+  "init"
+  "let"
+  "new"
+  "return"
+  (inflight_specifier)
+] @keyword
+
+(identifier) @variable
+(reference_identifier) @variable
+(member_identifier) @property
```

Comment:
Specifically, I believe these should be at the _top_ of the file here.

Reply:
I committed this change, will need to research further if that alone is not sufficient